snap/local/runtime-helpers/bin/renew_mgmt_token.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -e
```

Comment:
Couldn't we reuse this script and call it from the consul wait script. Obviously the Docker file would have to copy it to the folder that is volume mounted.

Reply:
never seen done this way before; i thought snap is a separate world by itself. we haven't done sharing a file between a snap and a docker

Reply:
This script is duplicate code for what you have in the consul wait script so the Docker file just needs to copy this file to the shared location next to the consul wait script.

Reply:
they are not exactly the same: 
- the snap is running as systemd service and thus I remove all the date-time logging information from the script eg. $(date)
- the renewToken in snap is not running as background process as it is running as a separate service here.

so we cannot just copy the exact file.

Reply:
But, will little refactoring it could be. It is 99% duplicate code.

Reply:
not sure how you can do refactoring?  remember every line of `echo $(date) .....` is different from the two.

Reply:
I am sure you can figure out a way ;-)

Reply:
it is kind of kludge solution refactoring either way IMO.  I think the better solution is to code this function in a security-bootstrapper subcommand and both snap and docker world can share. I'll rework on that to reduce the duplication concern you have.